Library/Formula/joshua.rb

```diff
@@ -0,0 +1,68 @@
+class Joshua < Formula
+  desc "A statistical machine translation decoder."
```

Comment:
Please remove the indefinite article from the description: `"Statistical machine translation decoder."`.